### DISCUSSION TITLE
Example DAG for New Azure Data Factory Hooks

### DISCUSSION THREAD

**Discussion by ShawnMcGough** [apache/airflow#14749] [Ideas] [answered]
The [Data Factory hooks](https://github.com/apache/airflow/blob/99c74968180ab7bc6d7152ec4233440b62a07969/airflow/providers/microsoft/azure/hooks/azure_data_factory.py) are new with [apache-airflow-providers-microsoft-azure 1.2.0](https://pypi.org/project/apache-airflow-providers-microsoft-azure/1.2.0/), but there are no examples or documentation on how to use them.

Would it be possible to have an example DAG added to the [example_dags ](https://github.com/apache/airflow/tree/master/airflow/providers/microsoft/azure/example_dags) directory of the azure providers? 🤔

And/or update the docs to include example usage?

**Comment by turbaszek** [comment, marked as answer]
@ShawnMcGough it would be awesome if you would like to contribute them! Having examples definitely helps our users 👍

**Reply by ShawnMcGough** [reply]
@turbaszek  I assure you that if I had a working DAG, I'd share it! 😅 I'm one of those new users that needs help. I have spend a few hours trying to get it to work. I was hopeful someone had a working example.

**Reply by turbaszek** [reply]
@ShawnMcGough got it, how about creating a feature request? I think we can mark it as a good first issue.
https://github.com/apache/airflow/issues/new/choose

**Reply by ShawnMcGough** [reply]
Ok, great. Thanks! Created here #14758
